be/src/vec/exec/scan/vscan_node.h

```diff
@@ -102,6 +102,35 @@ class ScannerDelegate {
     ScannerDelegate(ScannerDelegate&&) = delete;
 };
 
+class RunningScanner {
```

Comment:
Remove this class to scanncontext. It should not belong to scannode, because pipeline operator is not scannode.

And rename this class to ScanTask